builtin/providers/clc/resource_clc_server.go

```diff
@@ -44,15 +44,14 @@ func resourceCLCServer() *schema.Resource {
 				Type:     schema.TypeInt,
 				Required: true,
 			},
+			// optional
 			"password": &schema.Schema{
 				Type:     schema.TypeString,
-				Required: true,
+				Optional: true,
 			},
-			// optional
 			"description": &schema.Schema{
 				Type:     schema.TypeString,
 				Optional: true,
-				Default:  "",
```

Comment:
I think you should add Computed: true as well, since it can be generated by the platform. Similar to what is done for "private_ip_address."